src/modules/cpu/host_fused_functions.hpp

```diff
@@ -90,7 +90,7 @@ RppStatus color_twist_host(T* srcPtr, RppiSize srcSize, T* dstPtr,
         dstPtrTempB = dstPtr + (2 * imageDim);
 
         Rpp64u bufferLength = srcSize.height * srcSize.width;
-        Rpp64u alignedLength = bufferLength & ~3;
+        Rpp64u alignedLength = (bufferLength / 4) * 4;
```

Comment:
bufferLength & ~3 looks better option which avoids division

Reply:
Okay, 
I have reported to Abishek about it and got it resolved